### PR TITLE
Close chrome tab on mac after successful login

### PR DESCRIPTION
**Pull request by thiagomurakami**
#### What is the purpose of this pull request?
Add script to close Chrome tab that was successful on login. Only works on mac.

#### What problem is this solving?
Tab that is left open on mac is annoying.

#### How should this be manually tested?
?

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: scripts/closeChrome.scpt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+						tell w to activate
+
+						set (active tab index of window j) to i
+						return
+					end if
+				end repeat
+			end repeat
+		end tell
+	end if
+end run
```
Comment (pedro823) [inline scripts/closeChrome.scpt]: is... this... appleScript??
Will this run only on macs? What about windows/linux users?

File: src/modules/auth/login.ts [hunk 2]
```diff
@@ -92,9 +107,10 @@ export default async (options) => {
   const account = optionAccount || (usePrevious && cachedAccount) || await promptAccount(cachedAccount && optionWorkspace)
   const workspace = optionWorkspace || (usePrevious && cachedWorkspace) || 'master'
   try {
-    const { login, token } = await authAndSave(account, workspace, optionWorkspace)
+    const { login, token, returnUrl } = await authAndSave(account, workspace, optionWorkspace)
     log.debug('Login successful', login, account, token, workspace)
     log.info(`Logged into ${chalk.blue(account)} as ${chalk.green(login)} at workspace ${chalk.green(workspace)}`)
+    closeChromeTabIfMac(returnUrl)
```
Comment (pedro823) [inline src/modules/auth/login.ts]: would really appreciate other OSes too.

File: scripts/closeChrome.scpt (hunk 1 above)
Comment (thiagomurakami) [reply scripts/closeChrome.scpt]: Yes :O
I couldn't do this by javascript on `auth-server` 😢 , so the alternative was opening a PR here...
I don't even have any windows or linux environment configured to develop for those operating systems :(

Do you use linux? Want to help me on this one? :)

Comment (pedro823) [reply scripts/closeChrome.scpt]: I'll try doing this later in bash. Don't know if it'll work, though.

Comment (pedro823) [reply scripts/closeChrome.scpt]: Oh. Turns out `pkill --newest chrome` should do, IF the person didn't open any new tabs while logging in.

**Comment by firstdoit** [comment]
Or @pedro823 can you test whether pkill works on linux and mac (and firefox)

**Pull request merged**